prestoadmin/prestoclient.py

```diff
@@ -310,3 +325,21 @@ def _get_private_key(self, keystore):
             error('No alias %s found in %s. Available aliases are %s' %
                   (alias, self.coordinator_config.get_client_keystore_path(),
                    all_keys))
+
+    @staticmethod
+    def create_auth_headers(user, password):
+        if not user:
+            error('LDAP user (taken from %s in %s on the coordinator) cannot be null or empty' %
+                  (LDAP_CLIENT_USER_KEY, os.path.join(REMOTE_CONF_DIR, CONFIG_PROPERTIES)))
+            return {}
+        if not password:
+            error('LDAP password (taken from %s in %s on the coordinator) cannot be null or empty' %
+                  (LDAP_CLIENT_PASSWORD_KEY, os.path.join(REMOTE_CONF_DIR, CONFIG_PROPERTIES)))
+            return {}
+
+        if ':' in user:
+            error("LDAP user cannot contain ':': %s" % user)
```

Comment:
from @arhimondr -- "Check that user and password are not empty, and that username does not contain the : character."